primitive-types/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [dependencies]
 fixed-hash = { version = "0.6", path = "../fixed-hash", default-features = false }
-uint = { version = "0.8.1", path = "../uint", default-features = false }
+uint = { version = "0.8.3", path = "../uint", default-features = false }
```

Comment:
`uint` is a dev dep for `impls/serde` too; maybe worth setting the version there too?